corgi/core/models.py

```diff
@@ -912,7 +942,11 @@ class Namespace(models.TextChoices):
     # TODO: Or just switch from GenericForeignKey to regular ForeignKey?
     cnodes = GenericRelation(ComponentNode, related_query_name="%(class)s")
     software_build = models.ForeignKey(
-        SoftwareBuild, on_delete=models.CASCADE, null=True, related_name="components"
+        SoftwareBuild,
+        on_delete=models.CASCADE,
+        null=True,
+        related_name="components",
+        db_column="software_build_uuid",
```

Comment:
@JimFuller-RedHat Can you comment on this? If you change the primary key of a table, does Postgres automatically update all the ForeignKeys that link to that table?

I didn't try just updating the primary key. I did however do some research on it, and found that using the SQL as in migration 0058 was an approach used by others.

Reply:
I tried it out, and got an error during the migration at step 9.

```
django.db.utils.InternalError: cannot drop constraint core_softwarebuild_pkey on table core_softwarebuild because other objects depend on it
DETAIL:  constraint core_component_software_build_id_4f87341b_fk_core_soft on table core_component depends on index core_softwarebuild_pkey
```